feature.py

```diff
@@ -0,0 +1,24 @@
+import pandas as pd
+from sklearn.model_selection import train_test_split
+from sklearn.linear_model import LogisticRegression
+from sklearn.metrics import accuracy_score
+import unittest
+
+def func(a, b, c=0.25, d=42):
```

Comment:
This function fits data in LogisticRegression model and calculates an accuracy. Shall we rename function to be more explicit?
Also argument names could be more human readable? Shall we rename them as well? 

wdyt?
```suggestion
def fit_and_evaluate(X, prediction_column, dataset_split_ratio=0.25, random_seed=42):
```